Sources/Shared/UI/Documents/FileActionsViewController.m

```diff
@@ -314,8 +319,10 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 			
 		}
 		else if (row == 4)
-		{            
-			[fileActionsDelegate deleteFile:_file.path];
+		{
+            self.fileDeleteConfirmationBox = [[UIAlertView alloc] initWithTitle:Localize(@"Delete") message:Localize(@"Do you really want to delete this item") delegate:self cancelButtonTitle:Localize(@"Cancel") otherButtonTitles:Localize(@"Delete"), nil];
```

Comment:
Add a new line for the label:
"Confirm Delete" = "Are you sure you want to delete this item ?"